packages/sdk/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -46,53 +48,87 @@ export interface MedianRate {
   rate: BigNumber
 }
 
+export interface PairWithIdentifier {
+  pair: string
+  identifier: Address
+}
+
+export type ReportTarget = CeloToken | PairWithIdentifier
```

Comment:
`CeloContract` is a string enum, so anywhere `CeloContract.StableToken` is used, you get the string `'StableToken'`.
This pattern allows the user to refer to the thing by name rather than deal with an address. What do you think of instead of requiring the user to construct a `PairWithIdentifier`, that gets handled in `toCurrencyPairIdentifier` and the user can directly pass in the human-readable string?

To be more concrete, this line would change to:
```typescript
export type ReportTarget = CeloToken | string
```

Example usage:
```typescript
sortedOracles. getRates('CELO/BTC')
```

Reply:
I'm not entirely against this but I don't like the fact that we don't really get any type hinting / type safety.
All the methods would essentially take in any string and function without really make it it clear what is expected of that type. Plus doing `CeloToken | string` doesn't really tell you anything about what that type represents.

I agree that there's a bit more ceremony involved in this solution but we're more explicit about the intent behind the operations and what values can be passed to the methods.

Reply:
Totally fair! I agree `string` is a bit weird, though I think the ease of use is important.

What about having another enum for supported pairs? It would be even easier to use than strings, and allow for type safety.
The downside is that we'd have to update the enum every time a new pair is added, but maybe that's ok since adding a new pair to SortedOracles is a bit of a Process with governance and whatnot.

That said, I'm happy to approve this as-is and you can decide whether to go with my suggestion.